src/native/external/zlib-ng/CMakeLists.txt

```diff
@@ -183,8 +183,8 @@ elseif(MSVC)
     # (who'd use cmake from an IDE...) but checking for ICC before checking for MSVC should
     # avoid mistakes.
     # /Oi ?
-    set(WARNFLAGS /W3)
-    set(WARNFLAGS_MAINTAINER /W4)
+    # set(WARNFLAGS /W3)
+    # set(WARNFLAGS_MAINTAINER /W4)
```

Comment:
I was unable to disable in zlib-ng.cmake the error `D9025: overriding '/W4' with '/W3'`, seems that this gets overriden somewhere else, but couldn't find where. By commenting these lines, I was able to get the build to succeed. But ideally I would like to avoid modifying zlibn-ng files and figure out why this isn't working.

Reply:
Thanks @jkoritzinsky for fixing this in  https://github.com/dotnet/runtime/pull/102403/commits/20caea1c001152bd44e3847deb86aa8fb8575967